client/src/themes/theme.ts

```diff
@@ -1,13 +1,45 @@
 import { createTheme } from '@mui/material/styles';
+import React from 'react';
+
+declare module '@mui/material/styles' {
+  interface TypographyVariants {
+    label: React.CSSProperties;
+  }
+
+  // allow configuration using `createTheme`
+  interface TypographyVariantsOptions {
+    label?: React.CSSProperties;
+  }
+}
+
+// Update the Typography's variant prop options
+declare module '@mui/material/Typography' {
+  interface TypographyPropsVariantOverrides {
+    label: true;
+  }
+}
 
 export const theme = createTheme({
   typography: {
-    fontFamily: '"Open Sans", "sans-serif", "Roboto"',
+    fontFamily: '"Roboto", "Open Sans", "sans-serif"',
     fontSize: 12,
     button: {
       textTransform: 'none',
       fontWeight: 700,
     },
+    h1: {
+      fontSize: 30,
+      paddingBottom: 20,
```

Comment:
Let's only resort to applying font sizes, weight or casing to variants. We can use CSS classes to apply padding and margins as they may differ for each use case